playbook/app/pb_kits/playbook/pb_icon/docs/_icon_default.jsx

```diff
@@ -1,13 +1,29 @@
-import React from 'react'
+import React from "react"
 
-import Icon from '../_icon'
+import Icon from "../_icon"
+import {Caption} from "../.."
 
 const IconDefault = (props) => {
   return (
     <div>
-      <Icon
-          fixedWidth
-          icon="user"
+      <Icon fixedWidth
+          icon='angles-down'
+          {...props}
+      />
+      <Icon fixedWidth
+          icon='circle-arrow-right'
+          {...props}
+      />
+      <Icon fixedWidth
+          icon='arrow-circle-right'
+          {...props}
+      />
+      <Caption
+          marginY='md'
+          text='Font Awesome (no alias & not in our Playbook-icons lib)'
+      />
+      <Icon fixedWidth
+          icon='elephant'
```

Comment:
This test was only a **sanity** check to prove that if the icon and the alias does not exist, then it will still render FA for now.